.travis.yml

```diff
@@ -46,6 +46,7 @@ install:
 script:
   - if [[ $TEST_COVERAGE == 'true' ]]; then composer test-coverage ; else composer test ; fi
   - if [[ $CS_CHECK == 'true' ]]; then composer cs-check ; fi
+  - if [[ $TEST_COVERAGE == 'true' ]]; then composer static-analysis ; fi
```

Comment:
Just noticed that this runs when `$TEST_COVERAGE == ' true'` - perhaps it should have its own flag? (unsure about upstream guide)

Reply:
Thats how it was implemented in laminas/laminas-cli#39 which was used by @weierophinney during feedback regarding the step-by-step guide. 🤷‍♂️